application/common/models/User.php

```diff
@@ -142,6 +139,7 @@ public static function findOrCreate($username = null, $email = null, $employeeId
         $user = self::findOne(['employee_id' => $personnelUser->employeeId]);
         if ( ! $user) {
             $user = new User();
+            $user->uuid = $personnelUser->uuid;
```

Comment:
will `uuid` be a strict API for "personnel" systems?  I ask because I was thinking the pattern used for these personnel systems was an adapter.  If so, the interface would need to identify this I would think.  For example, let's say we adapt to a personnel system that uses `id` instead of `uuid`, would this be a concern?

Reply:
That's a very good question. I've been assuming that there is only ever one personnel system (id-broker) because Id Sync handles that abstraction. If that's not the case, I think there's also a problem with other features like "hide" (Do Not Disclose). @fillup what are your thoughts?

Reply:
yeah, I think that's right, we'd never "adapt" to something other than broker here...that makes sense.

Reply:
So I wonder if we still need the `Interface`. Is this an opportunity to remove code, or is it a case of "if it's not broken don't fix it"?

Reply:
I would remove it if I knew we needed to constrain ourselves to the broker.